etc/kayobe/environments/ci-multinode/README.md

```diff
@@ -0,0 +1,48 @@
+# Multinode Test Environment 
+
+## Set up hosts
```

Comment:
I guess this is how you did it for your testing? There is actually some terraform to automate this. Perhaps @jackhodgkiss or @grzegorzkoper could add some info on how to use that.